dealership/vehicles.py

```diff
@@ -1,15 +1,36 @@
+multiplier = {'sale': {'car': 1.2 , 'motorcycle' : 1.1, 'truck': 1.6}, 'purchase': {'car': 0.004 , 'motorcycle': 0.009, 'truck': 0.02}}
```

Comment:
put this inside Vehicle class instead cause this dictionary technically belongs to Vehicle .. if you're using `self.__class__`, your dictionary will be `{'sale': {'Car': 1.2, 'Motorcycle':1.1, 'Truck':1.6}}`